Makefile

```diff
@@ -56,14 +56,7 @@ libs:
 
 .PHONY : bin
 bin: libs
```

Comment:
Was this change on purpose? Does bin not need any of these other directories to be built? It looks like the build is passing so I guess?

Reply:
Note that "bin" is calling "libs" ("bin: libs"), therefore all the internal libs are already compiled there. the code I removed was meaningless (compiling the same libraries again).